src/main/java/org/roda_project/commons_ip2/validator/component/structuralMapComponent/StructuralMapComponentValidator.java

```diff
@@ -673,15 +673,27 @@ private ReporterDetails validateCSIP88(final MetsValidatorState metsValidatorSta
         final DivType div = struct.getDiv();
         if (div != null && div.getLABEL().equals("CSIP")) {
```

Comment:
Instead of checking the division for the label here the current `struct`(Map) needs to be checked whether it has a label and if it is "CSIP" above.